### DISCUSSION TITLE
Installation Instructions

### DISCUSSION THREAD

**Discussion by JordNaN** [sensmetry/sysml-2ls#7] [Q&A]
I cannot run a .sysml file after following the steps outlined in the "Quick Start" section of the ReadMe. To run code written in VSCode, the application requires a debugger, which is not mentioned in the installation instructions for this extension. Is there a specific debugger to install or instructions to follow to run .sysml files and enable the features presented in the ReadMe?

**Comment by daumantas-kavolis-sensmetry** [comment]
All the features are enabled simply by opening a `.sysml` or `.kerml` file. The extension works by parsing all `.sysml` and `.kerml` files in the workspace and sending relevant information to VS Code, no code running required. The features provided by this extension are similar to those of other extensions for programming languages excluding debuggers.
